src/Sylius/Bundle/ProductBundle/Resources/config/doctrine/model/Product.orm.xml

```diff
@@ -63,6 +63,7 @@
             <gedmo:timestampable on="update"/>
         </field>
         <field name="deletedAt" column="deleted_at" type="datetime" nullable="true" />
+        <field name="enabled" column="enabled" type="boolean" />
```

Comment:
1. Convention is to keep default values in models, not the database.
2. Nullable = false by default in Doctrine.